Source/WebCore/crypto/keys/CryptoKeyRSAComponents.h

```diff
@@ -72,6 +72,7 @@ class CryptoKeyRSAComponents {
     virtual ~CryptoKeyRSAComponents();
 
     Type type() const { return m_type; }
+    bool isPublic() const { return m_type == Type::Public; }
```

Comment:
This function doesn't add anything that type() doesn't already have.

Reply:
removed.